src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -0,0 +1,116 @@
+
+  handleTap = (context: any, href: string) => {
+    SwitchBoard.presentNavigationViewController(context, href)
+  }
+
+  getFeaturedArtistEntityCollection = (
+    artists: FeaturedArtists_collection["artworksConnection"]["merchandisableArtists"]
+  ) => {
+    return artists.map((artist, index) => {
+      const hasArtistMetaData = artist.nationality && artist.birthday
+      return (
+        <Box width="100%" key={index} pb={20}>
+          <TouchableWithoutFeedback onPress={() => this.handleTap(this, `/artist/${artist.slug}`)}>
+            <EntityHeader
+              imageUrl={artist.image.resized.url}
+              name={artist.name}
+              meta={hasArtistMetaData ? `${artist.nationality}, b. ${artist.birthday}` : undefined}
+              href={`/artist/${artist.slug}`}
+              FollowButton={
+                <Button
+                  variant={artist.isFollowed ? "primaryBlack" : "secondaryOutline"}
+                  size="small"
+                  longestText="Following"
+                >
+                  {artist.isFollowed ? "Following" : "Follow"}
+                </Button>
+              }
+            />
+          </TouchableWithoutFeedback>
+        </Box>
+      )
+    })
+  }
+
+  render() {
+    const artists = get(this.props, p => p.collection.artworksConnection.merchandisableArtists)
+    if (!artists) {
+      return null
+    }
+
+    const hasMultipleArtists = artists.length > 1
+
+    const artistCount = 3
+    const remainingCount = artists.length - artistCount
+    const truncatedArtists = this.getFeaturedArtistEntityCollection(artists).slice(0, artistCount)
+    const headlineLabel = "Featured Artist" + (hasMultipleArtists ? "s" : "")
+
+    return (
+      <Box pb={1}>
+        <Sans size="2" weight="medium" pb={15}>
+          {headlineLabel}
+        </Sans>
+        <Flex flexWrap="wrap">
+          {this.state.showMore || artists.length <= artistCount ? (
+            this.getFeaturedArtistEntityCollection(artists)
+          ) : (
+            <>
+              {truncatedArtists}
+              <TouchableHighlight
+                onPress={() => {
+                  this.setState({ showMore: true })
+                }}
+              >
+                <EntityHeader initials={`+ ${remainingCount}`} name="View more" />
+              </TouchableHighlight>
+            </>
+          )}
+        </Flex>
+      </Box>
+    )
+  }
+}
+
+export const CollectionFeaturedArtistsContainer = createFragmentContainer(FeaturedArtists, {
+  collection: graphql`
+    fragment FeaturedArtists_collection on MarketingCollection {
+      # TODO: size:9 is not actually limiting to 9 items. We need to figure out
+      #  why the back-end is not respecting that argument.
+      artworksConnection(aggregations: [MERCHANDISABLE_ARTISTS], size: 9, sort: "-decayed_merch") {
+        merchandisableArtists {
+          slug
+          internalID
+          name
+          image {
+            resized(width: 100) {
```

Comment:
Per [this comment](https://github.com/artsy/emission/pull/2001#discussion_r363786940) on the original PR, we should update this to be either 500, or the actual screen width if it's not too much work.